### PR TITLE
Show Real Post-Data Issue fixed

### LINKED ISSUE #9: Show Real Post-Data
### Instructions

`API Endpoint  = https://code-kitty-api.herokuapp.com/api/v1/post/users/all`

1. Create a Function in **Post Service Class** for fetching posts data.
2. Post Returns a Data type of` { success: boolean; data: Post[]; message: string }`
3. Implement a Function to Fetch Post Data and Assign it to an Array in `feed.component.ts`
4. Loop the post-card components with the response & show real data.

### PR DESCRIPTION
**Pull request by AashifAhamed**
#9 Fixed

### CODE REVIEW

File: src/app/components/feed/feed.component.ts [hunk 1]
```diff
@@ -9,7 +11,9 @@ import { PostFormComponent } from './post-form/post-form.component';
 })
 export class FeedComponent implements OnInit {
   loading: boolean = true;
-  constructor(public modalService: NgbModal) {}
+
+  posts:Post[] = [];
+  constructor(public modalService: NgbModal,public postservice: PostService) {}
 
   ngOnInit(): void {}
```
Comment (Safnaj) [inline src/app/components/feed/feed.component.ts:18]: call the getAllPosts() method inside ngOnInit()

File: src/app/components/feed/feed.component.ts [hunk 2]
```diff
@@ -19,4 +23,23 @@ export class FeedComponent implements OnInit {
       backdrop: 'static',
     });
   }
+
+  getAllPosts() { 
+      this.postservice.getAllPostsData().subscribe((data) => {
+        this.posts = data;
+    });
```
Comment (Safnaj) [inline src/app/components/feed/feed.component.ts:30]: API call in two places. Remove this.

**Review by Safnaj — CHANGES_REQUESTED** [review]
Changes requested

File: src/app/components/feed/feed.component.html [hunk 3]
```diff
@@ -29,9 +29,20 @@ <h6 class="text-black-50 mt-2">Any tips on your mind..?</h6>
           </button>
         </div>
         <app-loader *ngIf="!loading; else postFeed"></app-loader>
+
         <ng-template #postFeed>
-          <app-post-card></app-post-card>
+          <div class="row" *ngIf="filterdPost" >
```
Comment (Safnaj) [inline src/app/components/feed/feed.component.html:34]: `filteredPost` should be changed to `posts`

File: src/app/components/feed/feed.component.html [hunk 4]
```diff
@@ -29,9 +29,20 @@ <h6 class="text-black-50 mt-2">Any tips on your mind..?</h6>
...
+            <div *ngFor="let post of filterdPost;let i = inndex">
```
Comment (Safnaj) [inline src/app/components/feed/feed.component.html:35]: `filteredPost` should be changed to `posts`
Spelling mistake on `inndex`

**Review by Safnaj — CHANGES_REQUESTED** [review]
Changes requested

**Comment by Shehanka** [comment]
@AashifAhamed please have a look, CI is failing https://github.com/codekittyshow/code-kitty/runs/4247864593?check_suite_focus=true

**Pull request closed without merging**